### PR TITLE
setup for master tags

### PR DESCRIPTION
**Pull request by jyasveer**

### CODE REVIEW

File: f8a_worker/storages/s3_userintent.py [hunk 1]
```diff
@@ -9,3 +9,16 @@ def store_in_bucket(self, input_json):
...
+    def store_master_tags(self, input_json):
+        if 'ecosystem' not in input_json or 'data' not in input_json:
+            raise ValueError("Ecosystem and data are needed in supplied JSON, got "
+                             "{} instead".format(input_json))
+        self.store_dict(input_json['data'], input_json['ecosystem'] + '/manual_tag_list.json')
+
+    def fetch_master_tags(self, ecosystem):
+        if not ecosystem:
+            raise ValueError("Ecosystem is needed to fetch the master tag list")
+        file_name = ecosystem + '/manual_tag_list.json'
```
Comment (jpopelka) [inline f8a_worker/storages/s3_userintent.py]: `file_name = os.path.join(ecosystem, 'manual_tag_list.json')`

Comment (jyasveer) [reply f8a_worker/storages/s3_userintent.py]: thanks. let me fix this.

Comment (jyasveer) [reply f8a_worker/storages/s3_userintent.py]: this is not the actual path. this is just a prefix which acts as a path for the json file in the bucket

Comment (jpopelka) [reply f8a_worker/storages/s3_userintent.py]: ok 👍

**Comment by msrb** [comment]
[test]

**Comment by jpopelka** [comment]
[test]

File: f8a_worker/dispatcher/nodes.yml [hunk 2]
```diff
@@ -477,7 +477,7 @@
       import: 'f8a_worker.storages'
       configuration:
         <<: *configurationS3
-        bucket_name: '{DEPLOYMENT_PREFIX}-bayesian-user-intent'
+        bucket_name: '{DEPLOYMENT_PREFIX}-manual-tags'
```
Comment (fridex) [inline f8a_worker/dispatcher/nodes.yml]: Do we want to reuse the old bucket? Wouldn't it be better to create a new adapter for this purpose?

Comment (jyasveer) [reply f8a_worker/dispatcher/nodes.yml]: this is a bucket which we are not using anymore, so i updated it to use in this crowd sourcing work flow.

**Pull request merged**